airflow/jobs.py

```diff
@@ -562,8 +561,6 @@ def __init__(
         :param processor_poll_interval: The number of seconds to wait between
             polls of running processors
         :type processor_poll_interval: int
-        :param run_duration: how long to run (in seconds) before exiting
```

Comment:
Same.

Is this a part of this PR? This is already a part of https://github.com/apache/incubator-airflow/pull/4320

Reply:
Good catch, thanks